src/Database/Redis/Sentinel.hs

```diff
@@ -42,19 +42,20 @@ module Database.Redis.Sentinel
   ) where
 
 import           Control.Concurrent
-import           Control.Exception     (Exception, IOException, evaluate, throwIO)
+import           Control.Exception      (Exception, IOException, evaluate, throwIO)
 import           Control.Monad
-import           Control.Monad.Catch   (Handler (..), MonadCatch, catches, throwM)
+import           Control.Monad.Catch    (Handler (..), MonadCatch, catches, throwM)
 import           Control.Monad.Except
-import           Data.ByteString       (ByteString)
-import qualified Data.ByteString       as BS
-import qualified Data.ByteString.Char8 as BS8
-import           Data.Foldable         (toList)
-import           Data.List             (delete)
-import           Data.List.NonEmpty    (NonEmpty (..))
-import           Data.Typeable         (Typeable)
+import           Control.Monad.IO.Class (liftIO)
+import           Data.ByteString        (ByteString)
+import qualified Data.ByteString        as BS
+import qualified Data.ByteString.Char8  as BS8
+import           Data.Foldable          (toList)
+import           Data.List              (delete)
+import           Data.List.NonEmpty     (NonEmpty (..))
+import           Data.Typeable          (Typeable)
```

Comment:
Wow, the file diff interface really cannot handle stuff getting spaced over a little bit.